RocketDataTests/CollectionDataProviderTests/SharedCollectionDataProviderTests.swift

```diff
@@ -781,7 +781,8 @@ class SharedCollectionDataProviderTests: SharedCollectionTests {
      This ensures that data providers are always in sync across cache loading boundries.
      */
     func testAddCollectionWhileCacheLoads() {
-        var finishCacheLoad = {}
+        var finishCacheLoad: (()->Void)?
+        let finishCacheLoadExpectation = expectationWithDescription("Wait for finishCacheLoad to be set")
```

Comment:
I think it would be clearer to name this `collectionForKeyCalledExpectation`

The current name makes it sound like the cache has finished loading, but it hasn't.